design-toolkit/components/src/components/loading-placeholder/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,53 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
I'm not a fan of snapshot testing. I'd prefer to remove this

Reply:
Seriously, we don't use snapshot testing?